infra/update_layers_to_latest.py

```
@@ -0,0 +1,374 @@
+                    continue
+
+                # Parse layer ARN
+                match = re.match(
+                    r'(arn:aws:lambda:[^:]+:\d+:layer:)([^:]+)(:)(\d+)$',
+                    layer_arn
+                )
+                if match:
+                    arn_prefix = match.group(1)
+                    layer_name = match.group(2)
+                    colon = match.group(3)
+                    version = int(match.group(4))
```

Ruff (0.14.4)

Local variable `version` is assigned to but never used

Remove assignment to unused variable `version`

(F841)